### PR TITLE
fixed image path to staff's photos

### PR DESCRIPTION
**Pull request by vietcent**
is that due to how ruby/rails/mvc works? only Sophia's image path (VP of Women in Technology) worked so I changed everyone's to match hers.

### CODE REVIEW

**Comment by vietcent** [comment]
Diego are you dead lol

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -5,6 +5,10 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class with default "from" parameter.
   config.mailer_sender = "pwreset@asudisc.org"
+  
+  # config secret key
+  config.secret_key = '46a8a008507d8c995c0487c0b2a2212dbb4244270e19df073941511456b84e
```
Comment (dioms) [inline config/initializers/devise.rb]: Haven't done Rails in a while so not super familiar with this. 
Did you update devise and then it forced you to add this secret key? If so, we'd want to make this an environment variable so we're not committing our secrets. I added a new secret key to Heroku called DEVISE_SECRET so you should be able to just replace this key with ENV["DEVISE_SECRET"], then set an environment variable on your local (whatever you want)

**Comment by dioms** [comment]
Lol, I appreciate all the awesome changes! Just had a comment on the devise secrets stuff, once that is changed I'll merge the pull request!

File: .gitignore [hunk 2]
```diff
@@ -7,9 +7,6 @@
 # Ignore bundler config
 /.bundle
```
Comment (dioms) [inline .gitignore]: You should add `node_modules` to your `.gitignore` so we don't add a ton of files to the repo

**Comment by vietcent** [comment]
yea my bad, that's why I closed the branch for a bit because of the secret keys. thanks

**Comment by vietcent** [comment]
alright i removed node_modules( eslint and jslint) and changed the devise secret key

**Comment by dioms** [comment]
Awesome :sparkles: :sparkles:.

**Comment by dioms** [comment]
You want to add travis ci??

**Comment by vietcent** [comment]
lmao i was just playing around with it at the moment. i branched it out to do it and i'll see if it works out

**Pull request merged**